crates/shim/Cargo.toml

```diff
@@ -24,6 +24,7 @@ serde = "1.0.136"
 uuid = { version = "0.8.2", features = ["v4"] }
 signal-hook = "0.3.13"
 oci-spec = "0.5.4"
+cgroups-rs = "0.2.8"
```

Comment:
`shim` is generic trait, we should not depend on linux specific concepts (like cgroups)

Reply:
OK